package.json

```diff
@@ -91,6 +91,13 @@
     "build:product": "ng build product --prod",
     "release:organization:with-changelog": "cd feature-libs/organization && release-it && cd ../..",
     "release:product:with-changelog": "cd feature-libs/product && release-it && cd ../..",
+    "build:misc": "yarn --cwd feature-libs/misc/storefinder run build:schematics && ng build misc --prod",
+    "release:misc:with-changelog": "cd feature-libs/misc && release-it && cd ../..",
+    "build:product-configurator": "yarn --cwd feature-libs/product-configurator run build:schematics && ng build product-configurator --prod",
+    "e2e:cy:run:product-configuration":"yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:product-configuration",
+    "e2e:cy:start-run:product-configuration":"start-server-and-test start http-get://localhost:4200 e2e:cy:run:product-configuration",
+    "e2e:cy:run:cpq":"yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:cpq",
+    "release:product-configurator:with-changelog": "cd feature-libs/product-configurator && release-it && cd ../..",
```

Comment:
Most of those commands were dropped in develop:
- we've dropped `misc` lib in favour of `storefinder`
- we've dropped `smoke` commands, because now all e2e run for each PR in CI - no need to distinct it anymore

Please leave only commands needed for your feature